linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -253,7 +241,7 @@ private[emitter] final class ClassEmitter(sjsGen: SJSGen) {
     } yield {
       val docComment = js.DocComment("@constructor")
       val ctorDef =
-        if (isJSClass) globalVarDef("c", className, ctorFun)
+        if (isJSClass) fileLevelVar("b", genName(className)) := ctorFun
```

Comment:
Isn't `typeVar` still visible here?
```suggestion
        if (isJSClass) typeVar := ctorFun
```